BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Views/SkontoProceedView.swift

```diff
@@ -8,7 +8,7 @@ import UIKit
 import GiniCaptureSDK
 
 class SkontoProceedView: UIView {
-    private lazy var payButton: MultilineTitleButton = {
+    lazy var proceedButton: MultilineTitleButton = {
```

Comment:
[Suggestion] lets keep it private and add action to init, something like:

```
private var proceedAction: (() -> Void)?`

    init(proceedAction: (() -> Void)?) {
        self.proceedAction = proceedAction
    }
```

The main goal not to add target on public button out of view, we will have target setup in view and will call injected action